src/web/routes/application/do-you-have-children-three-or-under/do-you-have-children-three-or-under.js

```diff
@@ -0,0 +1,32 @@
+const { validate } = require('./validate')
+const { YES } = require('../common/constants')
+
+const contentSummary = (req) => ({
+  key: req.t('doYouHaveChildrenThreeOrUnder.summaryKey'),
+  value: req.t(req.session.claim.doYouHaveChildrenThreeOrUnder)
+})
+
+const pageContent = ({ translate }) => ({
+  title: translate('doYouHaveChildrenThreeOrUnder.title'),
+  heading: translate('doYouHaveChildrenThreeOrUnder.heading'),
+  buttonText: translate('buttons:continue'),
+  yes: translate('yes'),
+  no: translate('no'),
+  explanation: translate('doYouHaveChildrenThreeOrUnder.explanation')
+})
+
+// Update true case to direct to enter children dob page
+const next = req => req.session.claim.doYouHaveChildrenThreeOrUnder === YES ? '/are-you-pregnant' : '/are-you-pregnant'
```

Comment:
For this PR should we just leave this as `const next = () => '/are-you-pregnant'`, looks a little confusing otherwise. We will obviously revisit it when we add the new page in.

Reply:
Updated to simplified method.